NEWS.md

```diff
@@ -0,0 +1,20 @@
+# Pandora 23.11.0
+
+## Features
+The main function that facilitate the data retrieval and aggregation from the API is:
+
+ - `getData()`
```

Comment:
Can't find this function - copy-paste mistake?

Reply:
No, this function is not yet implemented and will be part of 
- https://github.com/Pandora-IsoMemo/pandora-data/issues/3
Sorry, I should have mentioned this. :speak_no_evil: